### DISCUSSION TITLE
FullStack build failed

### DISCUSSION THREAD

**Discussion by callmeitachi** [DioxusLabs/dioxus#2410] [Q&A]
I love Dioxus so much!because it's syntax looks even more Rust.

I have the following code:
```rust
use dioxus::{html::data, prelude::*};
use gloo_net::http::Request;
pub fn Login() -> Element {
    let onsubmit=move |evt:FormEvent| async move {
        if let Ok(a)=get_data("http://127.0.0.1:5000".to_string()).await {
            todo!()
        };
    };
    rsx! {
        h1 {"Login"}
        form {
            onsubmit,
            input {
                r#type:"text",id:"username",name:"username"
            }
            label {"Username"}
            br {}
            input {
                r#type:"password",id:"password",name:"password"
            }
            label {"Password"}
            br {}
            button {"Login"}
        }
    }
}
#[server]
async fn get_data(url:String) -> Result<String,ServerFnError> {
    let res=reqwest::Client::new().get(url).send().await.unwrap();
    let data1=res.text().await.unwrap();
    Ok(data1)
}
```

it can't build when I run the ' dx build --features web ' command.

And what error about this command?
  "PS D:\rust_example\dioxus-motor> cargo run --features ssr
error: none of the selected packages contains these features: ssr, did you mean: web?"

**Comment by ealmloff** [comment]
You can build your fullstack application with `dx build` or `dx build --platform fullstack`. Your fullstack application should have two features like [this example](https://github.com/DioxusLabs/dioxus/blob/main/packages/fullstack/examples/axum-hello-world/Cargo.toml#L20-L21): `web` and `server`. `dx build` will build the application once for each feature to create both the server and client bundle
